compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.h

```diff
@@ -68,6 +68,10 @@ createLLVMCPUSynchronizeSymbolVisibilityPass();
 std::unique_ptr<InterfacePass<mlir::FunctionOpInterface>>
 createLLVMCPUTileAndFusePass(int64_t tilingLevel = -1);
 
+// Pass to Tile the Root Op and Fuse Producer and Consumer.
```

Comment:
Please add the definition of `Root` to the comment. I think it could be "the op that has lowering_config". If there are multiple root ops, then it failed (or it's undefined). (IMO, it is better to just fail.)